samples/tex2html.js

```diff
@@ -15,8 +16,7 @@ let html = MathJax.document('<html></html>', {
 
 MathJax.handleRetriesFor(() => {
 
-    html.TestMath(process.argv[3] || '').compile().typeset();
-    let math = html.math.pop();
+    let math = html.convert(process.argv[3] || '', {end: STATE.CONVERT});
```

Comment:
I think this should be `STATE.TYPESET`, as `STATE.CONVERT` stops before the `typeset()` call.